### ISSUE TITLE
Wiki + master-reference

### ISSUE THREAD

**Issue by rebeccaradding** [foundersandcoders/wiki#1]
Hi @ronanyeah, what is your vision for this wiki? 

Based on what I see, I'm wondering if this information could be merged into the master-reference. @bradreeder thoughts?

**Comment by ronanyeah** [comment]
@rebeccaradding Yes I totally agree. Basically the master reference is what I had in mind for the wiki to become. There's not too much information but I'll move over what there is.

**Comment by ronanyeah** [comment]
I think we should consider using the wiki section of the master reference as well as `.md` files. Pull requests are suitable for high importance sections like curriculum but it may be beneficial to have a more fluid information source for less important things like gitter room lists.

**Comment by iteles** [comment]
@ronanyeah I respectfully disagree with that. One of the big reasons for using github is that we have a historical record of how things have changed and evolved over time through the commits. That's both interesting and valuable.
If we start using the wiki, we lose that as well as the ability to tie changes to specific issues (i.e. to a specific need raised by someone which led to that change). We also put ourselves at the mercy of people making random calls on what is 'important'. 

cc also @bradreeder & @sofer who 'thumbs upped' the [above comment](https://github.com/foundersandcoders/wiki/issues/1#issuecomment-271568230) in case you want to disagree with me here 👍

**Comment by ronanyeah** [comment]
I think it could be useful as a 'workspace' for things that eventually will be transferred to the main repo or **scrapped**. Github wikis also do have history and simple version control. I mentioned it elsewhere but I also think all of these repos would benefit from using Github projects alongside issues too. It's like a trello plugin, it's really simple and could be a good way of isolating and combining tasks and measuring progress. Unless I'm missing the amazing power of labels, just using issues seems to require a crazy amount of reading and navigation.

**Comment by iteles** [comment]
Projects :+1:

**Comment by ronanyeah** [comment]
I've cloned the wiki contents so this can be deleted at any point :+1: 
@iteles @rebeccaradding @bradreeder

**Comment by iteles** [comment]
@sofer @rebeccaradding Who is responsible for the follow up of the issues you guys raised such as this one?

**Comment by rebeccaradding** [comment]
@iteles in this case, I would suggest that @ronanyeah delete it since he created it. What do you think?
